src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's postal code in the address book. Guarantees: immutable.
+ */
+public class PostalCode {
+    public static final String MESSAGE_POSTAL_CODE_CONSTRAINTS = "Postal code should only contain numbers";
+    public static final String POSTAL_CODE_VALIDATION_REGEX = "^\\d+$";
```

Comment:
postal code is generally 6 digits, no? :-D

you have set high expectations, so I will push you to do better every time, please don't get discouraged. :-)